dashboard/test/ui/features/feedback_tab.feature

```diff
@@ -31,8 +31,8 @@ Scenario: With dev flag, as teacher, tab is invisible when not reviewing student
   And element "#ui-test-feedback-time" does not exist
   And I press keys "Nice!" for element "#ui-test-feedback-input"
   And I press "#ui-test-submit-feedback" using jQuery
-  And I wait until ".editor-column" contains text "Nice!"
-  And element "#ui-test-feedback-time" contains text "Last updated"
+  And element ".editor-column" contains text "Nice!"
```

Comment:
Is it safe to no longer wait for this?

Reply:
Yes - the wait was from an earlier iteration where the text would disappear from the input area and re-appear in a preview area. Now, the preview area is gone and the text stays in the input area - so it should always be there and shouldn't depend on a wait.